superset/security/manager.py

```diff
@@ -864,7 +872,7 @@ def copy_role(
         self.get_session.commit()
 
     def set_role(
-        self, role_name: str, pvm_check: Callable[[PermissionView], bool]
+        self, role_name: str, pvm_check: Callable[[PermissionView], bool], pvms:[]
```

Comment:
Type should be `list[PermissionView]` here?

Reply:
Correct. Changes Done.